src/schemas/21-526EZ/schema.js

```diff
@@ -371,6 +393,34 @@ let schema = {
         }
       }
     },
+    privateRecordReleases: {
```

Comment:
These validations are probably all a good idea, but perhaps we should make a comment that none of these are actually required, or externally validated as this info will be sent as an ancillary form? We've discussed this a few times so perhaps future maintainers will benefit from this context.

Reply:
Good call